### DISCUSSION TITLE
About opening tickets to the public

### DISCUSSION THREAD

**Discussion by synuns** [mondaycom/vibe#2116] [Q&A] [answered]
I went through the source code with considerable interest and developed a lot of affection for vibe.
I'm interested in contributing, so I leave a question.

I noticed that the tickets are managed by monday.com internal services.

Realistically, since it's a company library, you can't manage all tickets publicly, but since it's open source, I'd like to ask if you'd consider managing them publicly.

**Comment by talkor** [comment, marked as answer]
Thank you so much for your kind words and your interest in contributing to our project. We're thrilled to hear that you have developed an affection for Vibe!

Regarding your question, you are correct that we manage our tickets internally using Monday.com's services. This approach helps us streamline our processes and maintain a smooth workflow. However, we are committed to keeping the project running openly and transparently.

To that end, we encourage contributors like you to open issues on our public repository. This allows everyone to express their opinions, share their thoughts, and contribute to the project's development. Your input is invaluable and helps shape the future of Vibe.

We truly appreciate your enthusiasm and look forward to your contributions. If you have any further questions or suggestions, please feel free to reach out.
